Vertical_Garden_Barometer/font/RacingSansOne-Regular-38.lbdf.license

```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2012, Pablo Impallari (www.impallari.com|impallari@gmail.com),Rodrigo Fuenzalida (www.rfuenzalida.com|hello@rfuenzalida.com), with Reserved Font Name Racing Sans.
```

Comment:
The license filename is wrong: `.lbdf.license` should be `.bdf.license`

Reply:
Thank you for catching that. Fixed with the latest commit.